gammapy/catalog/tests/test_hawc.py

```diff
@@ -0,0 +1,24 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import pytest
+from ..hawc import SourceCatalog2HWC
+from ...utils.testing import requires_data
+
+@pytest.fixture(scope='session')
+def hwc2_fhl():
+    filename='$GAMMAPY_EXTRA/datasets/catalogs/2HWC.ecsv'#TODO: Not sure whether this makes sence, __init__ of SourceCatalog2HWC 
```

Comment:
Indeed, you can remove the `filename` and just have a one-line `return SourceCatalog2HWC()`

Reply:
Fixed.